build-zlib_x64.cmd

```diff
@@ -0,0 +1,28 @@
+CALL "F:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" amd64
```

Comment:
This path doesn't exist on appveyor and breaks the build: https://ci.appveyor.com/project/ah-/librdkafka-build/build/0.9.2-ci-55

I think you could either try and use an existing pre-compiled zlib or at least make it run on the buildserver.